app/src/components/blocks/selectStage/SelectStage.test.tsx

```diff
@@ -341,7 +357,7 @@ describe('<UploadDocumentsPage />', () => {
         );
 
         it('renders link to PCSE that opens in a new tab', () => {
-            renderSelectStage(setDocumentMock);
+            render(<SelectStage setDocuments={setDocumentMock} uploadDocuments={() => {}} />);
```

Comment:
Minor change but could we not put these renders in a render function to avoid much of the implementation duplication used across this file?